README.md

```diff
@@ -151,7 +151,9 @@ acquire this dependency.
    done this before, you may be prompted to elevate.
 7. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL to this
    repository, typically `https://github.com/microsoft/STL`
-8. Choose the architecture you wish to build in the IDE, and build as you would any other project. All necessary CMake
+8. Optional: If you wish to enable tests add `-DBUILD_TESTING=TRUE` under "CMake command arguments" in the CMake
+   settings editor.

```

Comment:
Should we clarify here and on 174 that the `llvm-project` submodule must be checked out for this to work? (IIRC this is why we don't `BUILD_TESTING` by default.)

Reply:
Hmmm, maybe I should turn it on by default now. This PR removed the dependency on `llvm-project` at build/configure time. You just need the submodule to actually run the tests.